internal/authtoken/transport.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2014 The Go Authors. All rights reserved.
```

Comment:
Where is this sourced?

Reply:
Added a code comment with a permalink to the original file & brief description of changes